feature/session/src/main/java/io/github/droidkaigi/confsched2020/session/ui/widget/SessionsItemDecoration.kt

```diff
@@ -54,34 +65,93 @@ class SessionsItemDecoration(
                 }
             }
 
+            if (startDateTimeText.dateText != null) {
+                c.drawText(
+                    startDateTimeText.dateText.value,
+                    startDateTimeText.dateText.positionX,
+                    startDateTimeText.dateText.positionY,
+                    textPaint
+                )
+            }
+
             c.drawText(
-                startTimeText.value,
-                startTimeText.positionX,
-                startTimeText.positionY,
+                startDateTimeText.startTimeText.value,
+                startDateTimeText.startTimeText.positionX,
+                startDateTimeText.startTimeText.positionY,
                 textPaint
             )
         }
     }
 
-    private fun calcTimeText(position: Int, view: View): StartTimeText {
+    private fun calcDateTimeText(
```

Comment:
Great work. 👀

Reply:
I felt it would be nicer if this method could be split into `calcTimeText` and `calcDateText` with offset calculation when I glanced, but it would be kinda heavy work. So it's okay but we are welcoming additional improvements anytime.